content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,92 @@
+- VLAN remote mechanism: the [VPP Forwarder](https://github.com/networkservicemesh/cmd-forwarder-vpp) now supports use of VLAN as a remote mechanism.
+  This allows the physical network to be partitioned into vWires distinguished by VLAN tags and introduces a method for cluster breakout.  Please note this feature is alpha as CI driven examples are not yet available.
+- Configurable VXLAN port:
+  In some deployments, the default VXLAN port 4789 may already be used by the underlying infrastructure, thus making the routers/switches drop packets sent by NSM over the same port due to security policies.
+  NSM_VXLAN_PORT env variable is now added to the [VPP forwarder](https://github.com/networkservicemesh/cmd-forwarder-vpp), allowing users to set a different port number.
+
+
+### OVS Forwarder
+A new forwarder based on the OpenvSwitch (OVS) data plane has been added to NSM as alternative to the VPP forwarder. The [OVS forwarder](https://github.com/networkservicemesh/cmd-forwarder-ovs) can install its own OVS instance or re-use an existing OVS instance on the host. For standard use cases, the OVS forwarder supports the local kernel mechanism (using veth pairs) and remote VXLAN mechanism with typical kernel datapath performance.
+
+For high-performance use cases the [OVS forwarder](https://github.com/networkservicemesh/cmd-forwarder-ovs) can off-load the NSM data plane to a Mellanox ConnectX SmartNIC, relying on standard OVS HW-offload capabilities. In this scenario, the OVS forwarder injects ConnectX (smart) Virtual Functions into NSC and NSE pods as local mechanism, where they can be consumed as DPDK or RDMA user-space interfaces to achieve full pass-through performance. The SmartNIC handles bonding and VXLAN encapsulation transparently for the applications.
+
+The [OVS forwarder](https://github.com/networkservicemesh/cmd-forwarder-ovs) is also planned to support coming forwarder use cases, such as remote VLAN mechanism for break-out to external fabric, VLAN-trunking of multiple client connections on a single NSE interface and P2MP forwarding.
+
+[Try the OVS Forwarder](https://github.com/networkservicemesh/deployments-k8s/tree/v1.1.1/examples/ovs)
+
+
+### Usability
+
+- Examples in [deployment-k8s](https://github.com/networkservicemesh/deployments-k8s/blob/v1.1.1/README.md) are greatly simplified by using [Spire K8s Registrar](https://github.com/spiffe/spire/blob/4e1a60a4d065f6a028f8c384c85603fbc151d5e8/support/k8s/k8s-workload-registrar/README.md) to create workload identities.
+- Kustomize examples in [deployment-k8s](https://github.com/networkservicemesh/deployments-k8s/blob/v1.1.1/README.md) can now be run with urls rather than inline in a checked out copy of the repo as was required in v1.0.0
+
+### Stability
+- Healing - simplified to improve stability and reduce reconvergence time
+- Refresh, and Timeout vastly simplified resulting in improved stability.
```

Comment:
Remove the comma

Reply:
Fixed in https://github.com/networkservicemesh/site/pull/207/commits/a94b37d7475128c89c4e040f6aac0789ff094afa